parcels/kernels/__init__.py

```diff
@@ -0,0 +1 @@
+from .advection import *  # noqa
```

Comment:
No, since `advection.py` needs to on the PYHTONPATH, so if you move it outside the `parcels` directory/module it becomes a separate python package on it's own. Creating a separate sub-module is actually the whole point of separating kernels from the core engine, but the y should still be part of `parcels`.